pytype/file_utils_test.py

```diff
@@ -107,9 +107,15 @@ class TestExpandSourceFiles(unittest.TestCase):
       file_utils.replace_separator("foo/bar/baz/e.py")
   ]
 
+  SCRIPT_CODE = """
+  #!/usr/bin/python3
+  print("hello, world")
+  """
+
   def _test_expand(self, string):
     with test_utils.Tempdir() as d:
       fs = [d.create_file(f) for f in self.FILES]
+      fs += [d.create_file("my_script", self.SCRIPT_CODE)]
```

Comment:
What is this line accomplishing? The test still asserts that expand_source_files only finds the .py files.

Reply:
Previously when I was checking every file in the directory to check if it was a script, this line would have made sense to make sure we don't miss a script file. Now that we only check for a file that is passed to `pytype`, this line  looks redundant.

I will remove it. Thanks for pointing that out.